src/plugins/nest.js

```diff
@@ -0,0 +1,294 @@
+'use strict'
+
+function createWrapCreate (tracer, config) {
+  return function wrapCreate (create) {
+    return function createWithTrace () {
+      const instance = arguments[0]
```

Comment:
Why not specify these in the signature?

Reply:
It has an extremely long signature, so thought it best to adopt the pattern here: https://github.com/signalfx/signalfx-nodejs-tracing/blob/f3533da4c5a57df945e32e6aa0d3991ac6de4ca9/src/plugins/graphql.js#L12

Reply:
That's not what `normalizeArgs` is necessarily doing, since those arguments can be of different forms.  You can just put the parameters you will use: `createHandlerWithTrace(instance, ignored, methodName)`.  To me that's more clear than the devoting space to `arguments` access.  If you needed the last n args or something conditional then I think what you are doing makes more sense.